esp/platform/espcontext.hpp

```diff
@@ -136,9 +155,16 @@ esp_http_decl const unsigned int getTxSummaryGroup();
 esp_http_decl const unsigned int readTxSummaryGroup(const char* group);
 esp_http_decl bool getTxSummaryResourceReq();
 esp_http_decl unsigned getSlowProcessingTime();
+esp_http_decl LogMsgDetail mapLegacyEspLogLevelToJlogThreshold(LogLevel level);
 
+esp_http_decl LogMsgCategory getEspLogCategoryForContext(IEspContext* ctx, LogMsgCategory defaultLogMsgCategory);
```

Comment:
This should probably be a const reference to the category - I'm not sure how gcc will pass otherwise.